cli/src/commands/deduplicate.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs')
+const { reporter, chalk } = require('@dhis2/cli-helpers-engine')
+const makePaths = require('../lib/paths')
+const { listDuplicates, fixDuplicates } = require('../lib/yarnDeduplicate')
+
+const handler = async ({ cwd }) => {
+    const paths = makePaths(cwd)
+
+    if (paths.yarnLock === null) {
+        reporter.error('Could not find yarn.lock')
+        process.exit(1)
+    }
+
+    const yarnLock = fs.readFileSync(paths.yarnLock, 'utf8')
+    const deduped = fixDuplicates(yarnLock)
+    fs.writeFileSync(paths.yarnLock, deduped)
+
+    const duplicates = listDuplicates(deduped)
+    if (duplicates.size > 0) {
+        reporter.error('Failed to deduplicate the following packages:')
```

Comment:
Failing to deduplicate with `fixDuplicates` may occur if there are multiple major versions of a dependency.